### DISCUSSION TITLE
评论系统故障

### DISCUSSION THREAD

**Discussion by loikuan** [ywmoyue/biliuwp-lite#109] [Q&A] [answered]
无法点赞及回复评论

**Comment by ywmoyue** [comment]
哪个视频？还是说所有视频都这样

**Reply by loikuan** [reply]
全部都这样

**Reply by ywmoyue** [reply]
最新版本是否已恢复？

**Reply by loikuan** [reply]
沒

**Reply by ywmoyue** [reply]
请把日志文件通过邮箱发送给我看看，邮箱地址可以在uwp关于页找到

**Reply by loikuan** [reply]
发了

**Reply by ywmoyue** [reply]
收到了，我有空看一下

**Reply by ywmoyue** [reply]
怪了，看日志是Cookie的一个数据没了，但是登录状态却还在，麻烦试一下重新登录看会不会恢复，重新登录也不会恢复的话我后面出个手动添加这个数据的功能

**Reply by loikuan** [reply]
重新登录点赞评论又好了，好怪哟

**Reply by ywmoyue** [reply]
https://github.com/ywmoyue/biliuwp-lite/discussions/13#discussioncomment-6061866
@Ech023 发现Cookie会被垃圾清理软件清理掉，@loikuan 你看看有没有使用过类似的垃圾清理软件
顺便问一下你在这之前大概是从哪个版本开始一直保持着登录状态没有重新登录过

**Reply by loikuan** [reply]
我一直用的是火绒，刚才又清了一次垃圾，再打开没发现有问题，哪个版本开始登录不是很记得了，大概是bili-lite重新开始更新那时候吧，也就是老哥你接盘开始维护那时

**Comment by ywmoyue** [comment, marked as answer]
4.6.9 已变更Cookie存储方案，不再会发生Cookie被清除问题
